src/web/mjs/connectors/ComicoNovel.mjs

```diff
@@ -29,6 +29,10 @@ export default class ComicoNovel extends Connector {
                                 // TODO: include figcaption.cn-picture__caption
                             } else {
                                 element.parentElement.style.margin = '0';
+                                if (${Engine._settings.NovelDarkmode.value}){
```

Comment:
Is it safe to assume that
- none of the child elements are applying a different color (e.g. dark grey) which would become almost invisible with black background
- Embedded images will work well with black background

Reply:
1. fair
2. then don't use dark mode?

Reply:
Maybe add a note in the settimg description, that using dark mode will not work on embedded images